geoh5py/shared/entity.py

```diff
@@ -515,11 +518,29 @@ def remove_data_from_group(
     def save(self, add_children: bool = True):
         """
         Alias method of :func:`~geoh5py.workspace.Workspace.save_entity`.
-
+        WILL BE DEPRECATED AS ENTITIES ARE ALWAYS AUTOMATICALLY UPDATED.
         :param add_children: Option to also save the children.
         """
+        warnings.warn(
+            "Entity.save() is deprecated and will be removed in next versions.",
+            DeprecationWarning,
+        )
         return self.workspace.save_entity(self, add_children=add_children)
 
+    # todo: circular import if creating a new workspace
+    # def save_as(self, filepath: str, add_children: bool = True) -> Entity:
+    #     """
+    #     Save the entity in a new created workspace.
+    #     :param filepath: The path of the new workspace.
+    #     :param add_children: Option to also save the children.
+    #     :return: the entity in the new workspace
+    #     """
+    #     # create a new workspace
+    #     workspace = Workspace.create(filepath)
+    #
+    #     # save the entity in the new workspace
+    #     return workspace.save_entity(self, add_children=add_children)
+
```

Comment:
Ok well, looks like that's not possible without a major re-write of the class. Let's just remove save_as here then...

Reply:
removing the commented part of the code.